app/graphql/types/query_types.rb

```diff
@@ -34,5 +34,15 @@ def wells(plate_id: nil)
     def plates
       Plate.all
     end
+
+    # Ont
```

Comment:
Can you put more comments like this in further up?  i.e. `# Wells` above the well queries and `# Plates` above those.  Then change this one to `# Ont::Libraries`